core/templates/dev/head/profile/preferences.html

```diff
@@ -77,6 +77,28 @@ <h2 class="oppia-page-heading">Preferences</h2>
 
         <div role="form" class="form-horizontal">
           <div class="form-group">
+            <label class="col-lg-2 col-md-2 col-sm-2">Subject Interests</label>
+            <div class="col-lg-10 col-md-10 col-sm-10">
+              <div ng-if="hasPageLoaded">
+                <select2-dropdown
+                  item="$parent.subjectInterests"
+                  tag-mode="true" allow-multiple-choices="true"
+                  invalid-search-term-message="Add a new subject interest (using lowercase letters and spaces)..."
```

Comment:
So, there's something different about this field and what appears on the Exploration Settings page.

Try typing "<script>alert(3);</script>" on the exploration settings page; it just won't allow it, and will display an "Add a new tag (using lowercase letters and spaces)" message. However, doing this in the preferences page causes an alert to pop up (which is bad!)